tensorflow/compiler/tf2xla/kernels/gpu_tf_kernel_custom_call.cc

```diff
@@ -513,8 +513,7 @@ void GenericTfCallback(void* stream_handle, void** buffers, const char* opaque,
   }
 }
 
-XLA_REGISTER_CUSTOM_CALL_TARGET_WITH_SYM(kTfCallbackCustomCall,
-                                         GenericTfCallback, "CUDA");
+XLA_REGISTER_CUSTOM_CALL_TARGET_WITH_SYM(kTfCallbackCustomCall,GenericTfCallback,xla::kXlaCustomCallGPUPlatform);
```

Comment:
This has nothing to do with custom calls though, I've meant a generic placeholder to expand into either "CUDA" or "ROCM", because there's quite a lot of code doing that.